provider/win_driver_installer_test.py

```diff
@@ -563,3 +563,29 @@ def balloon_test(test, params, vm, balloon_test_win):
     error_context.context("Reset balloon memory...done", test.log.info)
     session.close()
     error_context.context("Balloon test done.", test.log.info)
+
+
+@error_context.context_aware
+def fwcfg_test(test, params, vm):
+    """
+    Check if the Memory.dmp file can be saved.
+
+    :param test: kvm test object
+    :param params: the dict used for parameters.
+    :param vm: VM object.
+    """
+    tmp_dir = params["tmp_dir"]
+    if not os.path.isdir(tmp_dir):
+        process.system("mkdir %s" % tmp_dir)
+    dump_name = utils_misc.generate_random_string(4) + "Memory.dmp"
+    dump_file = tmp_dir + "/" + dump_name
+
+    output = vm.monitor.human_monitor_cmd('dump-guest-memory -w %s'
+                                          % dump_file)
+    if output:
+        test.fail("Save dump file failed as: %s" % output)
+    else:
+        cmd = "ls -l %s | awk '{print $5}'" % dump_file
+        dump_size = int(process.getoutput(cmd))
+        if dump_size == 0:
```

Comment:
Hi @leidwang I hit a product issue before is that: windows generated a memory dump file, the size of file is large than 0, but the dump cannot be opened by windbg tool, so if here need to consider adding windbg check for the generated dump file?

Reply:
Sure.Will add a checkpoint to cover it.Thanks!